src/services/userService.js

```diff
@@ -55,6 +57,36 @@ const updateUser = async (
   });
 };
 
+const signup = async (nickname, height, weight, skeletalMuscleMass, goalWeight, bodyFat, age, gender, user) => {
+  await AppDataSource.transaction(async () => {
+    if (!user.isNew) throwError(401, "NOT_NEW_USER");
+
+    validateUserInfo(nickname, height, weight, skeletalMuscleMass, goalWeight, bodyFat, age);
+
+    const existingUser = await userDao.findUserByNickname(nickname);
+    if (existingUser) throwError(409, "DUPLICATED_NICKNAME");
+
+    const existingGender = await genderDao.findGenderByName(gender);
+    if (!existingGender) throwError(404, "GENDER_NOT_FOUND");
+
+    const startDate = new Date();
+    const formattedStartDate = startDate.toISOString().slice(0, 10);
+    const endDate = new Date(startDate);
+    endDate.setDate(startDate.getDate() + 7);
+    const formattedEndDate = endDate.toISOString().slice(0, 10);
+    const subscribe = await subscribeDao.createSubscribe(formattedStartDate, formattedEndDate);
+
```

Comment:
```suggestion

    const now = moment(new Date())
    const startDate = now.format('YYYY-MM-DD');
    const endDate = now.add(7, "days").format("YYYY-MM-DD");
    const subscribe = await subscribeDao.createSubscribe(startDate, endDate);

```

moment라는 라이브러리가 있는데, 도움이 될 것 같아 전달드립니다! 정확한 문법은 필요에 따라 수정해서 사용하시면 될 것 같아요 :)

Reply:
추가해서 사용했습니다..! 감사합니다